src/System.ServiceModel.Syndication/ref/System.ServiceModel.Syndication.netcoreapp.cs

```diff
@@ -7,8 +7,24 @@
 
 namespace System.ServiceModel.Syndication
 {
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public partial struct ParseDateTimeArgs
+    {
+        public string DateTimeString { get { throw null; } set { } }
+        public System.Xml.XmlQualifiedName ElemntQualifiedName { get { throw null; } set { } }
+    }
```

Comment:
Typo in the property name. Should be Elem**e**ntQualifiedName. The same is in ParseUriArgs